conjure-java-core/src/integrationInput/java/com/palantir/product/DoubleAliasExample.java

```diff
@@ -47,6 +47,11 @@ public static DoubleAliasExample of(int value) {
         return new DoubleAliasExample((double) value);
     }
 
+    @JsonCreator
+    public static DoubleAliasExample of(long value) {
+        return new DoubleAliasExample((double) value);
```

Comment:
I think we probably need to check if the cast would end up imprecise on this

Reply:
Maybe? If this is a double value that got encoded as an exact value it should be precisely representable. I can add an epsilon comparison if you want though.